### DISCUSSION TITLE
Complete Tank Betting Game more rich

### DISCUSSION THREAD

**Discussion by 0xNevo** [community/community#49452] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

First do you know solidity?

Second do you have knowledge of algorithm?

Third do you know react?

These three questions are needed.
Let's complete!!

**Comment by runesatsdev** [comment]
Hi.
Of course I know these tech, so if you need my help, you can ping me.
Best Regards.

**Reply by 0xNevo** [reply]
OK, and more.

**Reply by runesatsdev** [reply]
Specially .
At recent I developing website to use these tech.
plz join me.

**Reply by 0xNevo** [reply]
Well, and then.
go on.

**Reply by 0xNevo** [reply]
Do you know Solidity?
Yes, I do have knowledge of Solidity. It is a programming language used for writing smart contracts on the Ethereum blockchain.

Do you have knowledge of algorithms?
Yes, as an experienced developer, I have in-depth knowledge and understanding of various algorithms and their implementation in code.

Do you know React?
Yes, I am familiar with React, which is a popular JavaScript library used for building user interfaces. With React, one can create reusable UI components which allow for faster development and easier maintenance of web applications.

I hope this answers your questions! Let me know if there is anything else I can assist you with.
